### ISSUE TITLE
Login always returns cancel v5.2.3, ios 13 Xcode 11 beta 4

### ISSUE THREAD

**Issue by pattessm** [facebook/facebook-swift-sdk#495]
## Checklist

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-swift-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

Xcode Version: 11.0 Beta 3
iOS Version: 13 Beta 3
Swift Version: 5.1
Installation Platform & Version: CocoaPods

## Goals
Login With Facebook

## Expected Results

Perform login without canceling under safari or app

## Actual Results

Login always returns cancel in the console, no error

## Steps to Reproduce

Tap on the login button

## Code Samples & Details
AppDelegate :
```swift
import FBSDKCoreKit
import Firebase
func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
    FirebaseApp.configure()  
    return ApplicationDelegate.shared.application(application, didFinishLaunchingWithOptions: launchOptions)
}
func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
    return ApplicationDelegate.shared.application(app, open: url, options: options)
}
```

Login Button Code : 
```swift
import FBSDKLoginKit
import FacebookLogin
@objc func didTapFacebookButton() {
LoginManager().logIn(permissions: ["manage_pages", "publish_pages"], viewController: vc) { (result) in
            switch result {
            case .cancelled:
                print("LOGIN REQUEST CANCELED")
            case .failed(let e):
                print("REQ FAILED HERE IS ERROR: ", e.localizedDescription)
            case .success(granted: _, declined: _, token: let token):
                print("GOT THE TOKEN: ", token.tokenString)
            }
        }

}
```

Info.plist:
```swift
<key>CFBundleURLTypes</key>
<array>
    <dict>
        <key>CFBundleURLSchemes</key>
        <array>
            <string>MY_APP_ID</string>
        </array>
    </dict>
</array>
<key>FacebookAppID</key>
<string>MY_APP_ID</string>
<key>FacebookDisplayName</key>
<string>MY_APP_NAME</string>
<key>LSApplicationQueriesSchemes</key>
<array>
    <string>fbapi</string>
    <string>fb-messenger-share-api</string>
    <string>fbauth2</string>
    <string>fbshareextension</string>
</array>
<key>NSAppTransportSecurity</key>
<dict>
    <key>NSAllowsArbitraryLoads</key>
    <true/>
    <key>NSAllowsArbitraryLoadsForMedia</key>
    <true/>
    <key>NSAllowsArbitraryLoadsInWebContent</key>
    <true/>
    <key>NSExceptionDomains</key>
    <dict>
        <key>NSIncludesSubdomains</key>
        <true/>
        <key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
        <false/>
    </dict>
    <key>facebook.com</key>
    <dict>
        <key>NSIncludesSubdomains</key>
        <true/>
        <key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
        <false/>
    </dict>
    <key>fbcdn.net</key>
    <dict>
        <key>NSIncludesSubdomains</key>
        <true/>
        <key>NSThirdPartyExceptionRequiresForwardSecrecy</key>
        <false/>
    </dict>
</dict>
```

**Comment by marianoprettysimple** [comment]
I'm having the same issue with the **Objective-C SDK 5.3.0**, **Xcode 11 beta 5** (I'm writing this here as there's no Issues option on the ObjC SDK repo...). After calling `[FBSDKLoginManager logInWithPermissions:fromViewController:handler:]`, no login dialog is shown, no error, the result just comes back as canceled.

**Comment by simonlee2** [comment]
i'm also having this issue with **Objective-C SDK 5.3.0, Xcode 11 beta 5**

**Comment by GitTheCode** [comment]
I am also seeing the same issue with Objective-C 5.3.0 Xcode 11 Beta5
I have tried version 5.2.2, 5.2.3 and 5.3.0 as there are various reports of these working/not.  I am not using Carthage or CocoaPods - just linking directly with static libs.
5.2.2 is crashing with:
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[FBSDKBridgeAPI presentationAnchorForWebAuthenticationSession:]: unrecognized selector sent to instance 0x28273c1e0'

5.2.3 and 5.3.0 do not crash but logInWithPermissions:fromViewController:handler:] gets an immediate callback with isCancelled set to true, error is an 'implicit cancel' 

iOS13 release date is coming up fast guys - please get this fixed - Thanks!

**Comment by marianoprettysimple** [comment]
This behavior seems to be gone on iOS 13 beta 7.

**Comment by starzplay** [comment]
It still happens for me on iOS13 beta 7

**Comment by yasinturkdogan** [comment]
It still happens for me on iOS13 beta 7

**Comment by marianoprettysimple** [comment]
> It still happens for me on iOS13 beta 7

Yes, sorry, I made a mistake when testing. 

This should be fixed soon. Check out this issue in the Objective C repo: https://github.com/facebook/facebook-objc-sdk/issues/1013

**Comment by joesus** [comment]
This is fixed in facebook/facebook-objc-sdk#1013 . Currently this SDK will automatically pull the latest version of the ObjC SDK so reinstalling the dependencies (pod install / carthage update) should resolve your issue. Closing.

**Comment by HristiyanZahariev** [comment]
This bug still appears for me even when I updated to 5.6.0
